aws/client.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+import boto3
+
+from .logger import get_logger
+
+
+class AWSClientWrapper:
```

Comment:
@Kelton8Z I did a minor cleanup to the c;ient code [here](https://github.com/jina-ai/stress-test/blob/master/distributed/aws/client.py)